.chloggen/kubeletstats-feature-gate.yaml

```diff
@@ -0,0 +1,30 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Even tho it has been a while, this will be a very impactful change. Lets add something at the top of the component's README too.

Reply:
should be adapted